ngraph_onnx/onnx_importer/utils/binary.py

```diff
@@ -59,3 +62,71 @@ def broadcast_for_binary_operation(onnx_node, ng_inputs):  # type: (NodeWrapper,
     start_axis = onnx_node.get_attribute_value('axis')  # start of mutually equal shape
     right = ng.broadcast(right, left.shape, start_axis)
     return left, right
+
+
+def numpy_style_broadcast_for_binary_operation(onnx_node, ng_inputs):
+    # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
+    """
+    Cast shape of two nodes to make them compatible for an element-wise binary operation.
+
+    :param onnx_node: a wrapped ONNX node
+    :param ng_inputs: left and right node (inputs of the binary op)
+    :return: left and right node after broadcasting
+    """
+    left = ng_inputs[0]
```

Comment:
Maybe we could check whether there are two inputs?

Reply:
I think its unnecessary because this function is called only when we are sure(ONNX documentation) that op have 2 operands.